backend/ssm/ssm.go

```diff
@@ -30,11 +33,11 @@ func (b *Backend) Get(ctx context.Context, key string) ([]byte, error) {
 	return b.fromCache(ctx, key)
 }
 
```

Comment:
Same for this one (I know you didn't create this @rogpeppe, I just think it should be addressed)

Reply:
done